src/Plugin.php

```diff
@@ -129,6 +133,28 @@ public static function admin_enqueue_scripts() {
     wp_enqueue_style('showrooms-admin-css', static::getBaseUrl() . '/dist/styles/showrooms-admin.css');
   }
 
+  /**
+   * @implements enqueue_scripts.
+   */
+  public static function wp_enqueue_scripts() {
+    if (!is_product_category()) {
+      return;
+    }
+
+    $query = get_queried_object();
+    if (empty($query->slug)) {
+      return;
+    }
+
+    if ($query->taxonomy !== 'product_cat') {
+      return;
+    }
+
+    if (have_rows('field_showroom_window', 'product_cat_' . $query->term_id)) {
+      wp_enqueue_style('showrooms-css', static::getBaseUrl() . '/dist/styles/showrooms.css');
```

Comment:
1. wp_enqueue_style() is only about styles, so a suffix `-css` is superfluous.

1. You should specify the whole plugin namespace to prevent name clashes.  You may use `Plugin::PREFIX` for that.

1. The filename should be singular, too.  (But doesn't have to repeat the full namespace again; in fact, can also just be base.css or theme.css depending on what SMACSS layer the styles relate to.)